astropy/io/fits/fitstime.py

```diff
@@ -109,6 +114,49 @@ def _verify_global_info(global_info):
                 'the use of local time scales should be restricted to alternate '
                 'coordinates.'.format(global_info['TIMESYS']))
 
+    # Verify global location
+    obs_geo = {}
+
+    for idx, attr in enumerate(['OBSGEO-X', 'OBSGEO-Y', 'OBSGEO-Z']):
+        val = global_info.get(attr, None)
+        if val is not None:
+            obs_geo[idx] = val
+    if len(obs_geo.items()) == 3:
+        global_info['location'] = EarthLocation(*[obs_geo[i] for i in range(0,3)],
+                                                unit='m')
+    else:
+        for idx, attr in enumerate(['OBSGEO-L', 'OBSGEO-B', 'OBSGEO-H']):
+            val = global_info.get(attr, None)
+            if val is not None:
+                obs_geo[idx] = val
+        if len(obs_geo.items()) == 3:
+            global_info['location'] = EarthLocation.from_geodetic(*[obs_geo[i] for i
+                                                                    in range(0,3)])
+
+    # Verify global time reference
+    mjd_ref = global_info.get('MJDREF', None)
+    jd_ref = global_info.get('JDREF', None)
+    date_ref = global_info.get('DATEREF', None)
+
+    if mjd_ref is not None:
+        ref_value = mjd_ref
+        ref_format = 'mjd'
+    elif jd_ref is not None:
+        ref_value = jd_ref
+        ref_format = 'jd'
+    elif date_ref is not None:
+        ref_value = date_ref
+        ref_format = 'fits'
+    else:
+        ref_value = 0.0
+        ref_format = 'mjd'
+    global_info['ref_time_value'] = ref_value
+    global_info['ref_time_format'] = ref_format
+
+    # Verify global unit
+    global_info['format'] = UNIT_TO_FORMAT.get(global_info['TIMEUNIT'],
```

Comment:
See comment above - I don't understand when this ever would be important, and would suggest removing it.